examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,172 @@
+    "publisher": "Librivox",
+    "published": "2008-10-12",
+    
+    "modified": "2016-08-11T19:32:18Z",
+    
+    "belongs_to": {
+      "collection": "Math for All",
+      "series": {
+        "name": "Geometry Trilogy",
+        "position": 4
+      }
+    },
+    
+    "duration": 15153,
+    "type": "audio/mpeg",
+    "bitrate": 128,
+    "samplerate": "8.5",
+    "channels": 2,
+    
+    "schema:hasPart": [
+        {
+            "@type": "http://schema.org/VisualArtwork",
+            "schema:url": "cover.jpg",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://schema.org/Book",
+            "schema:url": "this_is_for_copyright_on_text_copy",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://bib.schema.org/Audiobook",
+            "schema:url": "this_is_for_copyright_on_performance",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        }
+    ],
+    
+    "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/",
+    
+    "schema:hasPart": [
+        {
+            "schema:url": "http://www.archive.org/download/flatland_rg_librivox/flatland_1_abbott.mp3",
+            "narrator": "Abe Narrator"
+        }
+    ],
+    "schema:hasPart": [
+        {
+            "schema:url": "http://www.archive.org/download/flatland_rg_librivox/flatland_2_abbott.mp3",
+            "narrator": "Beatrice Narrator"
+        }
+    ],
+},
+
+"links": [
+    {"rel": "self", "href": "https://github.com/HadrienGardeur/audiobook-manifest/blob/master/examples/flatland.audiobook-manifest", "type": "application/audiobook+json"},
+    
+    {"rel": "alternate", "href": "https://hadriengardeur.github.io/webpub-manifest/examples/MobyDick/manifest.json", "type": "application/webpub+json"},
+    {"rel": "alternate", "href": "https://[...]/appendix.pdf", "type": "application/pdf"},    
+    
+    {"rel": "license", "href": "http://[...]/license.lcpl", "type": "application/vnd.readium.lcp.license.v1.0+json"},    
+    
+    {"rel": "cover", "href": "http://ia800704.us.archive.org/9/items/LibrivoxCdCoverArt12/Flatland_1109.jpg", "type": "image/jpeg", "height": 300, "width": 300},
+    {"rel": "alternate", "href": "https://archive.org/download/flatland_rg_librivox/Flatland_64kb_librivox.m4b", "type": "audio/x-m4b", "bitrate": 64},
+    {"rel": "alternate", "href": "http://www.archive.org/download/flatland_rg_librivox/flatland_rg_librivox_64kb_mp3.zip", "type": "application/zip"},
+    {"rel": "alternate", "href": "http://www.archive.org/download/flatland_rg_librivox/flatland_rg_librivox_archive.torrent", "type": "application/x-bittorrent"}
+],
+
+"spine": [
+    {
+        "href": "http://www.archive.org/download/flatland_rg_librivox/flatland_1_abbott.mp3", 
+        "type": "audio/mpeg", 
+        "bitrate": 128, 
```

Comment:
We repeat `bitrate` but not `samplerate` and `channels`. Maybe we should avoid repeating any of these elements at a link level if they don't differ from the ones expressed in `metadata`?